extensions/search-npm/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "search-npm",
+  "title": "Search npm",
+  "description": "",
+  "icon": "command-icon.png",
+  "author": "mrmartineau",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search npm",
+      "subtitle": "Search for npm package information",
```

This fits better our UI guidelines: 
```suggestion
      "title": "Search Packages",
      "subtitle": "npm",
```